src/app/component/core/search-popup/search-popup.component.html

```diff
@@ -0,0 +1,57 @@
+<ng-template [ngIf]="isSearchClicked">
+  <div class="search-bar-wrapper">
+    <div class="center-flex-wrapper">
+      <div class="search-icon"></div>
+        <input class = "search-field"
+               type="text"
+               placeholder="Search"
+               (keyup)="onKeyUp($event)">
+      <div class="close-icon"
+           (click)="closeSearch()"></div>
+    </div>
+  </div>
+
+  <div class="page-wrapper">
+    <div class="search-content-wrapper">
+      <div class="remaining-items" *ngIf="inputValue != undefined && itemsFound != undefined">
+        {{ itemsFound }} items found
+      </div>
+      <ng-template [ngIf]="isNewsSearchFound || isTipsSearchFound">
+        <div class="content-news" *ngIf="isNewsSearchFound">
+          <a [routerLink]="['/news']"
+             (click)="closeSearch()">
+            <h3>News</h3>
+          </a>
+          <div class="flex-container">
+            <app-search-item *ngFor="let element of newsElements"
+                             [searchModel]="element"
+                             (closeSearch)="closeSearch()"></app-search-item>
+          </div>
+        </div>
+
+        <div class="content-tips" *ngIf="isTipsSearchFound">
+          <a [routerLink]="['/welcome']"
+             pageScroll
+             href="#tips"
+             [pageScrollDuration]="500"
+             (click)="closeSearch()">
+            <h3>Tips & Tricks</h3>
+          </a>
+          <div class="flex-container">
+
+          </div>
+        </div>
+
+        <a class="all-results-link">
+          See all results
+        </a>
+      </ng-template>
+
+      <ng-template [ngIf]="(isNewsSearchFound === false)">
```

Comment:
Please, rewrite it ([ngIf]=!isNewSearchFound)

Reply:
fixed